stable/collabora-code/README.md

```diff
@@ -0,0 +1,87 @@
+# Collabora CODE
+
+[Collabora](https://www.collaboraoffice.com/code/) is a online office suite.
+
+## Introduction
+
+This chart creates a single Collabora CODE Pod to run Collabora CODE suite, for example as integration together with nextcloud. Installation is based on the docker documentation [CollaboraDocker](https://www.collaboraoffice.com/code/docker/).
+
+For most easy integration it´s recommended to use cert-manager together with your favorite ingress controller to get a fully working, ssl-terminated Collabora CODE server.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
```

Comment:
Can you please update this to Kubernetes 1.9+ as that is when the `apps/v1` API used below became available.

Reply:
Yes, of course. I fixed the README.